src/sass/modules/_m-schemaform.scss

```diff
@@ -45,22 +45,29 @@ dl.review {
   .review-row {
     display: flex;
     justify-content: space-between;
+    flex-direction: column;
     border-top: 1px solid $color-gray-light;
     padding: 1.5rem;
     padding-left: 0;
     &> dd {
       font-weight: bold;
-      padding-left: 5px;
-      min-width: 100px;
       text-align: left;
     }
-    &> dt {
-      max-width: 60%;
-    }
     &> dt > p {
       margin-top: 0;
       margin-bottom: 0;
     }
+    @media (min-width: $small-screen) {
+      flex-direction: row;
+      &> dd {
+        padding-left: 5px;
+        min-width: 100px;
+      }
+      &> dt {
```

Comment:
It looks you are following an existing convention. Perhaps simply creating an issue for this is good enough here.